owasp-dependency-check-suppressions.xml

```diff
@@ -61,6 +68,35 @@
     <cve>CVE-2022-4245</cve>
   </suppress>
 
+  <suppress>
+    <notes><![CDATA[
+      file name: sketches-java-0.8.2.jar
+    ]]></notes>
+    <cve>CVE-2021-40531</cve>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+      file name: azure-core-1.45.1.jar
+    ]]></notes>
+    <cve>CVE-2023-36052</cve>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+      file name: azure-core-http-netty-1.13.11.jar
+    ]]></notes>
+    <cve>CVE-2023-36052</cve>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+      file name: azure-identity-1.11.1.jar
+    ]]></notes>
+    <cve>CVE-2023-36415</cve>
+    <cve>CVE-2023-36052</cve>
+  </suppress>
```

Comment:
it seems a bit odd that by fixing one CVE we would be introducing a bunch of dependencies with new CVEs. Do we understand why those dependencies are being pulled in now?

Reply:
These are not being added by the json-path version change. These JARs are already present in the master branch. Most of these are new CVEs, and it is possible that it was not present earlier in the owasp-checklist. They were already added before this PR here:
1. [ion-path](https://github.com/apache/druid/blob/5edfa9429f40bbddfd868bcd0d51f106702199d3/licenses.yaml#L2401) 
2. [azure-core](https://github.com/apache/druid/blob/5edfa9429f40bbddfd868bcd0d51f106702199d3/extensions-core/azure-extensions/pom.xml#L72)
3. [sketches-java](https://github.com/apache/druid/blame/5edfa9429f40bbddfd868bcd0d51f106702199d3/extensions-contrib/ddsketch/pom.xml#L38)
4. [azure-identity](https://github.com/apache/druid/blob/5edfa9429f40bbddfd868bcd0d51f106702199d3/extensions-core/azure-extensions/pom.xml#L56)
5. [azure-core-http-netty](https://github.com/apache/druid/blame/5edfa9429f40bbddfd868bcd0d51f106702199d3/licenses.yaml#L4262)